driver/src/contracts/snapp_contract.rs

```diff
@@ -276,20 +276,12 @@ impl SnappContract for SnappContractImpl {
         new_state: H256,
         deposit_hash: H256,
     ) -> Result<()> {
-        let account = self
-            .base
-            .account_with_sufficient_balance()
```

Comment:
So do we no longer ensure that the account has sufficient balance?

Reply:
The tx will fail if we don't have sufficient balance and it will say in the error message.